FWCore/Modules/src/PathActivityProbe.cc

```diff
@@ -0,0 +1,31 @@
+#include "DataFormats/Common/interface/PathActivityToken.h"
```

Comment:
I realize `PathActivityProducer` may not be the final name, but the file name should match the name of the producer class.